ingestion/src/metadata/ingestion/source/clickhouse_usage.py

```diff
@@ -130,7 +102,10 @@ def get_report(self):
         return self.report
 
     def close(self):
-        self.alchemy_helper.close()
+        pass
 
     def get_status(self) -> SourceStatus:
         return self.report
+
+    def test_connection(self) -> None:
+        pass
```

Comment:
Let's use the same test_connection approach as with the SQLSource, as we are using SQA engines